CefSharp.Example/AsyncBoundObject.cs

```diff
@@ -64,6 +64,10 @@ public JsSerializableClass[] ReturnClassesArray(string name)
             };
         }
 
+        public string[] ReceiveArray(string[] arg) {
```

Comment:
Braces on new line.

Probably be better if this method was called something like `EchoArray`